client/src/utils.ts

```diff
@@ -68,10 +75,30 @@ export function switchTheme(theme: string, set: (theme: string) => void) {
   }
 }
 
+export function isPayingSubscriber(user) {
+  const validSubscriptionTypes = [
+    MDN_PLUS_5M_PLAN_NAME,
+    MDN_PLUS_5Y_PLAN_NAME,
+    MDN_PLUS_10M_PLAN_NAME,
+    MDN_PLUS_10Y_PLAN_NAME,
+  ];
```

Comment:
These are **not** currently defined in `stage-build.yml` or `prod-build.yml`, so this function will always return false unless those env vars are set.

This explains why I'm not seeing the offline bar locally.